src/server/index.test.js

```diff
@@ -57,7 +57,11 @@ jest.mock('koa-socket', () => {
   return MockIO;
 });
 
-const game = Game({});
+const game = Game({
+  setup: () => {
+    return { seed: 0 };
```

Comment:
We should probably pass seed through a separate option in `Game` rather than through `G`.

Reply:
I was unsure about this one. The `seed` is now passed as separate option.